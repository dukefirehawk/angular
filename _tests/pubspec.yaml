name: _tests
description: Tests for AngularDart.
publish_to: none

environment:
  sdk: '>=3.6.0 <4.0.0'

dependencies:
  build_resolvers: ^2.1.0
  build_test: ^2.1.6
  collection: ^1.17.0
  logging: ^1.3.0
  ngcompiler: ^4.0.0-dev.1
  ngdart: ^8.0.0-dev.1
  ngtest: ^5.0.0-dev.1
  test: ^1.22.2
  web: ^1.1.1

dev_dependencies:
  analyzer: ^7.3.0
  build_runner: ^2.3.3
  build_web_compilers: ^4.1.3
  csslib: ^1.0.0
  lints: ^5.0.0
  meta: ^1.8.0
  mockito: ^5.3.2
  package_config: ^2.1.0
  source_span: ^1.9.1
  term_glyph: ^1.2.1
