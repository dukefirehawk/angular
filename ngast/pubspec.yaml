name: ngast
description: Parser and utilities for AngularDart templates
version: 3.0.0-dev.0
repository: https://github.com/angulardart-community/angular

environment:
  sdk: '>=3.6.0 <4.0.0'

dependencies:
  charcode: ^1.3.1
  collection: ^1.17.0
  meta: ^1.8.0
  source_span: ^1.9.1
  string_scanner: ^1.2.0

dev_dependencies:
  lints: ^5.0.0
  path: ^1.8.3
  test: ^1.22.2
