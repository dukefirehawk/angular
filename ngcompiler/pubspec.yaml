name: ngcompiler
description: >
  The AngularDart compiler.
  It compiles your component files written in Dart into templates so that you
  don't have to worry about all the complicated hassle.
version: 4.0.0-dev.1
repository: https://github.com/angulardart-community/angular

environment:
  sdk: '>=3.6.0 <4.0.0'

dependencies:
  analyzer: ^7.3.0
  args: ^2.3.2
  build: ^2.3.1
  build_test: ^2.1.6
  code_builder: ^4.4.0
  collection: ^1.17.0
  csslib: ^1.0.0
  dart_style: ^3.0.1
  logging: ^1.1.0
  meta: ^1.8.0
  ngast: ^4.0.0-dev.1
  ngdart: ^9.0.0-dev.1
  package_config: ^2.1.0
  path: ^1.8.3
  source_gen: ^2.0.0
  source_span: ^1.9.1

dev_dependencies:
  lints: ^5.0.0
  test: ^1.22.2
