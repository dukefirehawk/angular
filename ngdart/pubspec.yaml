name: ngdart
description: >-
  Fast and productive web framework by Google.
  Separate, but similar to the JavaScript framework with a similar name.
version: 8.0.0-dev.2
repository: https://github.com/angulardart-community/angular

environment:
  sdk: '>=3.6.0 <4.0.0'

dependencies:
  analyzer: ^7.3.0
  build: ^2.3.1
  built_collection: ^5.1.1
  build_modules: ^5.0.12
  built_value: ^8.4.3
  collection: ^1.17.0
  intl: ^0.20.0
#  js: ^0.7.2
  meta: ^1.8.0
  ngast: ^3.0.0-dev.0
  ngcompiler: ^3.0.0-dev.1
  stream_transform: ^2.1.0
  web: ^1.1.1

dev_dependencies:
  lints: ^5.0.0
