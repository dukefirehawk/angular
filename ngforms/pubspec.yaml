name: ngforms
description: >
  Forms framework for AngularDart.
  Easily create forms to accept input from users.
version: 6.0.0-dev.1
repository: https://github.com/angulardart-community/angular

environment:
  sdk: '>=3.6.0 <4.0.0'

dependencies:
  meta: ^1.8.0
  ngdart: ^9.0.0-dev.1
  web: ^1.1.1
dev_dependencies:
  build_runner: ^2.3.3
  build_test: ^2.1.6
  build_web_compilers: ^4.1.3
  lints: ^5.0.0
  mockito: ^5.3.2
  ngtest: ^6.0.0-dev.1
  test: ^1.22.2
