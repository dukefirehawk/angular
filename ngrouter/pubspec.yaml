name: ngrouter
description: >
  Router for AngularDart.
  It enables navigation from one view to the next as users perform tasks.
version: 5.0.0-dev.1
repository: https://github.com/angulardart-community/angular

environment:
  sdk: '>=3.6.0 <4.0.0'

dependencies:
  collection: ^1.17.0
  ngdart: ^8.0.0-dev.1
  web: ^1.1.1
dev_dependencies:
  async: ^2.10.0
  build_runner: ^2.3.3
  build_test: ^2.1.6
  build_web_compilers: ^4.1.3
  lints: ^5.0.0
  mockito: ^5.3.2
  ngtest: ^5.0.0-dev.1
  test: ^1.22.2
