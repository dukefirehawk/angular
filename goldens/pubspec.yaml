name: goldens
description: Goldens for AngularDart.
publish_to: none

environment:
  sdk: '>=3.6.0 <4.0.0'

dependencies:
#  js: ^0.7.2
  ngdart: ^8.0.0-dev.1

dev_dependencies:
  build_runner: ^2.3.3
  build_web_compilers: ^4.1.3
  dart_style: ^3.0.1
  glob: ^2.1.1
  lints: ^5.0.0
  path: ^1.8.3
